docs/_docs/energy.md

```diff
@@ -303,19 +303,36 @@ while zero for $r>r\_c+w\_c$.
 ### Assorted Short Ranged Potentials
 
 The potentials below are often used to keep particles apart and/or to introduce stickiness.
-The atomic interaction parameters, $\epsilon\_i$, are taken from the topology according to the naming scheme
-shown in the table.
+The atomic interaction parameters, e.g., $\sigma_i$ and $\epsilon\_i$, are taken from the
+topology.
 
-type           | $\epsilon$       | $u(r)$
--------------- | ---------------- | --------------------------------------------------------
-`hardsphere`   |                  | $\infty$ for $r < \sigma\_{ij}$
-`hertz`        | `eps_hertz`      | $\epsilon\_{ij} \left ( 1-r / \sigma\_{ij}\right )^{5/2}$ for $r<\sigma\_{ij}$
-`lennardjones` | `eps`            | $4\epsilon\_{ij} \left ( (\sigma\_{ij}/r\_{ij})^{12} - (\sigma\_{ij}/r\_{ij})^6\right )$
-`squarewell`   | `eps_sw`         | $-\epsilon\_{ij}$ for $r<\sigma\_{ij}$ 
-[`wca`](http://dx.doi.org/ct4kh9) | `eps` | $u\_{ij}^{\text{LJ}} + \epsilon\_{ij}$ for $r < 2^{1/6}\sigma\_{ij}$
+type           | atomic parameters | $u(r)$ (non-zero part)
```

Comment:
please check that `make manual` passes (pdf creation)

Reply:
Errors corrected in the force-push. Just for convenience Charter instead of Garamond font was used.